sys/cbor/cbor.c

```diff
@@ -24,8 +24,8 @@
 #include <stdlib.h>
 #include <string.h>
 
-/* Automatically enable/disable ENABLE_DEBUG based on CBOR_NO_PRINT */
-#ifndef CBOR_NO_PRINT
+/* Automatically enable/disable ENABLE_DEBUG based on MODULE_CBOR_PRINT */
+#ifdef MODULE_CBOR_PRINT
```

Comment:
in my use case I want to use the `cbor_stream_decode` from the shell so I decided to use `printf` and not let this function be based on log or debug.

If you don't call `cbor_stream_decode` or `cbor_steam_print` the cbor module won't print anything.